README.md

```diff
@@ -70,7 +70,7 @@ Unofficial extensions for other frameworks and libraries are also available:
 
 * [Phony](https://github.com/eloquent/phpstan-phony)
 * [Prophecy](https://github.com/Jan0707/phpstan-prophecy)
-* [Laravel](https://github.com/Weebly/phpstan-laravel)
+* [Laravel](https://github.com/nunomaduro/larastan)
```

Comment:
Replacing someone's extension like this looks like a very offensive step to me.
cc @twistor

Reply:
I'm fine with this. I don't think I submitted it in the first place, since it wasn't really ready yet. High hopes and not enough time.